Sources/App/Migrations/071/UpdateVersionResetProductDependenciesWithDefault.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Fluent
+import SQLKit
+
+
+struct UpdateVersionResetProductDependenciesWithDefault: AsyncMigration {
+    func prepare(on database: Database) async throws {
+        // re-create field without default
+        try await database.schema("versions")
+            .deleteField("product_dependencies")
+            .update()
+        try await database.schema("versions")
+            .field("product_dependencies",
+                   .array(of: .json))
+            .update()
```

Comment:
Instead of altering the column and keeping the values, I'm now resetting the values, because they're tainted with the previous ambiguous processing.